frontend/src/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetailsView.tsx

```diff
@@ -43,6 +44,7 @@ const ModelVersionDetailsView: React.FC<ModelVersionDetailsViewProps> = ({
   const modelArtifact = modelArtifacts.items.length ? modelArtifacts.items[0] : null;
   const { apiState } = React.useContext(ModelRegistryContext);
   const storageFields = uriToStorageFields(modelArtifact?.uri || '');
+  const [rm] = useRegisteredModelById(mv.registeredModelId);
```

Comment:
We should pull in loaded, loadError and refresh from this fetch hook and make sure we handle them properly.

We can create composite variables for:
```ts
const loaded = modelArtifactsLoaded && registeredModelLoaded;
const loadError = modelArtifactsLoadError || registeredModelLoadError;
const refresh = () => {
  refreshModelArtifacts();
  refreshRegisteredModel();
};
```

and then use those any place we currently use `modelArtifactsLoaded`, `modelArtifactsLoadError` and `refreshModelArtifacts` here.